test/unit/deployment.bats

```diff
@@ -82,6 +82,23 @@ load _helpers
     [ "${actual}" = "128Mi" ]
 }
 
+@test "controller/Deployment: default resources for job" {
```

Comment:
I wonder if this should be combined with the above "default resources for controller" test, like how the "can set resources for controller" checks both job and controller.

Reply:
Fixed in 21c5dc2